partner_communication_switzerland/models/res_partner_zoom_attendee.py

```diff
@@ -0,0 +1,102 @@
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import models, fields
+
+
+COLOR_MAPPING = {
+    "invited": 0,
+    "confirmed": 10,
+    "declined": 1,
+    "attended": 11
+}
+
+
+class ZoomAttendee(models.Model):
+    _name = "res.partner.zoom.attendee"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+    _description = "Zoom Attendee"
+    _rec_name = "partner_id"
+    _order = "id desc"
+
+    partner_id = fields.Many2one("res.partner", "Partner", required=True, index=True)
+    zoom_session_id = fields.Many2one(
+        "res.partner.zoom.session", "Zoom session", required=True, index=True)
+    state = fields.Selection([
+        ("invited", "Invited"),
+        ("confirmed", "Confirmed"),
+        ("attended", "Attended"),
+        ("declined", "Declined")
+    ], default="invited")
+    optional_message = fields.Text()
+    color = fields.Integer(compute="_compute_color")
+
+    _sql_constraints = [
+        ("unique_participant", "unique(partner_id,zoom_session_id)",
+         "This partner is already invited to this zoom session!")
+    ]
+
+    def _compute_color(self):
+        for attendee in self:
+            attendee.color = COLOR_MAPPING.get(attendee.state)
+
+    def inform_about_next_session(self):
+        for attendee in self:
+            next_zoom = attendee.zoom_session_id.get_next_session()
```

Comment:
Maybe the method name is not very good because it's not at this time that we inform him. We just take note that he wants to be informed. Here we don't need to care about his language.

Reply:
I'm surprised because you used a context before : https://github.com/CompassionCH/compassion-switzerland/pull/1356/files#diff-56fb2534043f2802633676aad3d448b0aed27b7477958b255247be02335d34a5R617

and the function that look over available zoom meeting use lang as a search field